protos/flyteidl/admin/common.proto

```diff
@@ -237,14 +237,46 @@ message Annotations {
     map<string, string> values          = 1;
 }
 
+// Security context fields to define privilege and access control settings
+// In the future a mode (e.g. OVERRIDE, APPEND, etc) can be defined
+// to specify how to merge security context defined at registration and execution time.
+message SecurityContext {
+    // Map of custom annotations to be applied to the execution resource.
+    map<string, string> values          = 1;
+}
+
+// Config values to define privilege and access control settings
+// In the future a mode (e.g. OVERRIDE, APPEND, etc) can be defined
+// to specify how to merge configs defined at registration and execution time.
+message Config {
+    // Map of custom annotations to be applied to the execution resource.
+    map<string, string> values          = 1;
```

Comment:
can we call this field annotations then?

Reply:
sorry. wrong comment